core/dbt/task/run.py

```diff
@@ -400,7 +400,7 @@ def safe_run_hooks(
                     thread_id="main",
                     timing=[],
                     message=f"{hook_type.value} failed, error:\n {exc.msg}",
-                    adapter_response=exc.msg,
```

Comment:
The `BaseResult` was added here recently as part of #6059.  My understanding is that the adapter response isn't important.  Because i added types, this was failing since `exc.msg` is a string but `adapter_response` is a `Dict[str, Any]`.

Reply:
I think there was some conversation recently about the adapter_response, but I'm pretty vague on what was discussed. I think it often doesn't actually have any content, but maybe should?  But in any case assigning a string to it is clearly wrong.